ci/common/build.sh

```diff
@@ -7,27 +7,26 @@ build_make() {
 }
 
 build_deps() {
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
     DEPS_CMAKE_FLAGS="${DEPS_CMAKE_FLAGS} ${CMAKE_FLAGS_32BIT}"
   fi
-  if [[ "${FUNCTIONALTEST}" == "functionaltest-lua" ]]; then
+  if test "x${FUNCTIONALTEST}" = "xfunctionaltest-lua" ; then
```

Comment:
I have not seen anything advanced in our scripts. I could convert all of them to be POSIX-compatible, just only needed those from ci/common. Most code is in ci/common in any case and it was easy to convert.